### ISSUE TITLE
No Widgets displaying in Jupyter Lab

### ISSUE THREAD

**Issue by rakhusur** [JuliaGizmos/WebIO.jl#202]
I hope I didn't miss something but I followed the setup steps from the readme and `jupyter labextension list` gives me
```
JupyterLab v0.34.9
Known labextensions:
   app dir: /home/moritz/.local/share/jupyter/lab
        julia-web-io v0.1.0  enabled  OK*

   local extensions:
        julia-web-io: /home/moritz/.julia/packages/WebIO/GwCk0/assets/webio
```
but nothing appears when I do `using WebIO; node(:div, "Hello")`. I've noticed the following warning in the Browser console when launching jupyter lab:
```
WebIO cannot find webio/dist/bundle.js, so it cannot load the webcomponents-lite support. Widgets may be broken in this environment.
```

**Comment by shashi** [comment]
That's very strange, I tested #199 on the same version of JupyterLab. The fastest way to fix this maybe to get #200 to work.

Thanks!

**Comment by travigd** [comment]
Closing; Jupyter lab should work in master.
